### PR TITLE
Wiki boost

### PR DESCRIPTION
**Pull request by khushmeeet**
## Describe your Arc Boost
Simplify Wikipedia page for reading. Removes sidebar and increases font size.

## Checklist before requesting a merge

- [x] I have copied the extension zip file into the public/extensions folder

- [x] I have created a new JSON file in the data/manifest folder, with the extensions' id
- [x] I have filled all the blank lines in the JSON file
- [x] I have tested it locally
- [x] I have uploaded a picture here, then used the `user-images.githubusercontent...` URL inside the JSON file

> Upload your preview picture here
SCR-20221007-lfa

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -30,5 +30,7 @@
...
     "svelte.config.*": "*.env, .babelrc*, .codecov, .cssnanorc*, .env*, .htmlnanorc*, .mocha*, .postcssrc*, .terserrc*, api-extractor.json, ava.config.*, babel.config.*, cssnano.config.*, cypress.json, env.d.ts, htmlnanorc.*, jasmine.*, jest.config.*, jsconfig.*, karma*, playwright.config.*, postcss.config.*, puppeteer.config.*, svgo.config.*, tailwind.config.*, tsconfig.*, tsdoc.*, unocss.config.*, vitest.config.*, webpack.config.*, windi.config.*",
     "remix.config.*": "*.env, .babelrc*, .codecov, .cssnanorc*, .env*, .htmlnanorc*, .mocha*, .postcssrc*, .terserrc*, api-extractor.json, ava.config.*, babel.config.*, cssnano.config.*, cypress.json, env.d.ts, htmlnanorc.*, jasmine.*, jest.config.*, jsconfig.*, karma*, playwright.config.*, postcss.config.*, puppeteer.config.*, remix.*, svgo.config.*, tailwind.config.*, tsconfig.*, tsdoc.*, unocss.config.*, vitest.config.*, webpack.config.*, windi.config.*"
-  }
+  },
+  "workbench.colorTheme": "Vitesse Dark",
+  "workbench.preferredDarkColorTheme": "Vitesse Dark"
```
Comment (cristicretu) [inline .vscode/settings.json:35]:
```suggestion
  },
```

Comment (cristicretu) [inline .vscode/settings.json:35]:
```suggestion
  }
```

**Review by cristicretu — CHANGES_REQUESTED** [review]
I suggest adding `!important` for every rule, otherwise not all the will styles work. (or at least they didn't work from the start for me)
image

File: .vscode/settings.json (hunk 1 above)
Comment (khushmeeet) [reply .vscode/settings.json]: I updated the files as requested.

**Comment by cristicretu** [comment]
Great job! Cool boost and thanks for the first contribution 🥳

**Review by cristicretu — APPROVED** [review]

**Pull request merged**